index.bs

```diff
@@ -1454,7 +1454,7 @@ authenticator over a set of contextual bindings. Each attestation statement form
     [=hash of the serialized client data=], and returns either:
     - An error indicating that the attestation is invalid, or
     - The attestation type, and the trust path of the attestation. This trust path is either empty (in case of
-        self-attestation), a DAA root key (in the case of Direct Anonymous Attestation), or a set of X.509 certificates.
+        self-attestation), an identifier of a [=DAA Issuer public key=] (in the case of Direct Anonymous Attestation), or a set of X.509 certificates.
```

Comment:
the string used in [[FIDOEcdaaAlgorithm]], for `[=DAA Issuer public key=]`, is "ECDAA-Issuer public key".  We should replicate that herein, yes?

Are all the references to "Direct Anonymous Attestation (DAA)" herein actually in regards to "elliptic curve DAA" ?  if so, that should be made clear, eg using "ECDAA" where we use "DAA" (regardless of case), yes?